frontend/app/src/components/dashboard/NftBalanceTable.vue

```diff
@@ -31,19 +34,17 @@ const {
   fetchData,
   setPage,
   setOptions
-} = usePaginationFilters<NonFungibleBalance>(
-  null,
-  false,
-  useEmptyFilter,
-  fetchNonFungibleBalances,
-  {
-    extraParams,
-    defaultSortBy: {
-      key: 'name',
-      ascending: [true]
-    }
+} = usePaginationFilters<
+  NonFungibleBalance,
+  NonFungibleBalancesRequestPayload,
+  NonFungibleBalance & { lastPrice?: any }
```

Comment:
Since we are re-using this in multiple places maybe it would make sense to just extract it?